### PR TITLE
776 - app menu example

### LINKED ISSUE #776: IdsApplicationmenu: Misaligned footer in app menu
**Describe the bug**
Misaligned footer in app menu

**To Reproduce**
1. Go https://main.wc.design.infor.com/ids-app-menu/example.html
2. Open the App Menu
3. Notice the footer looks off and Misaligned
4. The footer looks off and misaligned

**Expected behavior**
Footer should not look misaligned 

**Version**

- ids-enterprise: [e.g. v4.9.0 or v4.10.0]

**Screenshots**
https://www.screencast.com/t/5Iu3wsVO

**Platform**
 - Infor Application/Team Name: [WC IDS New Web Components]
 - OS Version: [e.g. Windows 10]

**Additional context**
None

Should look more like: https://main-enterprise.demo.design.infor.com/components/applicationmenu/example-personalized-role-switcher.html

### PR DESCRIPTION
**Pull request by celhalal**
**Explain the details for making this change. What existing problem does the pull request solve?**
Aligned header and footer icons in app-menu.

**Related github/jira issue (required)**:
closes #776

**Steps necessary to review your pull request (required)**:
1. Checkout branch, npm run start
2. Go to http://localhost:4300/ids-app-menu/example.html
3. Click icon menu (top left corner)
4. Check the misaligned footer
    -  example.html changes:
    - changed the Infor logo from a div to a slot element 
    - removed the toolbar footer and replaced it with the slot's name for the logo
5. Created another demo called example-footer.html
    Go to http://localhost:4300/ids-app-menu/example.html
     - this demo only includes the toolbar footer so I removed the infor logo
     - removed type='fluid' and align='center-even' since it was pushed to the right.

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

File: src/components/ids-app-menu/demos/index.yaml [hunk 1]
```diff
@@ -8,6 +8,9 @@
   - link: example.html
     type: Main Example
     description: Shows a basic app menu
+  - link: example-footer.html
+    type: Main Example
```
Comment (andriivarhanov) [inline src/components/ids-app-menu/demos/index.yaml:12]: please change to just Example since there is main example above

**Review by andriivarhanov — COMMENTED** [review]
looks good, just a minor text adjustment

**Review by tmcconechy — CHANGES_REQUESTED** [review]
Actually another suggestion it looks like some of the changes effected the masthead and toolbar:

https://percy.io/00aaa841/enterprise-wc/builds/19725648/changed/1108099133?browser=chrome&browser_ids=23&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=375&widths=375%2C1280 

https://percy.io/00aaa841/enterprise-wc/builds/19725648/changed/1108088263?browser=chrome&browser_ids=23&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=1280&widths=375%2C1280

Probably becauseof https://github.com/infor-design/enterprise-wc/pull/809/files#diff-a5dee3d127a457aaf5d9d5a79e725cddd541907baafb6b0c646b677afd91f44eR22 so maybe we need to add that rule either in the app-menu scss rules or as a setting (can youo make it it work for only .fluid?)

**Comment by tmcconechy** [comment]
@celhalal looking at the screen shots in percy it seems like it still breaks:

http://localhost:4300/ids-masthead/example.html
and
http://localhost:4300/ids-toolbar/formatter.html

But i dont understand 100% why? Can compare to:
https://main.wc.design.infor.com/ids-masthead/example.html
https://main.wc.design.infor.com/ids-toolbar/formatter.html

**Review by tmcconechy — APPROVED** [review]
I changed this a bit to isolate the styles for fluid/center-even

**Pull request merged**